plutus-cbor/test/Plutus/Codec/CBOR/EncodingSpec.hs

```diff
@@ -71,46 +139,53 @@ propCompareWithOracle ::
   x ->
   Property
 propCompareWithOracle encodeOracle encodeOurs x =
-  ( convert (CBOR.toStrictByteString oracle) === ours
+  ( Plutus.toBuiltin (CBOR.toStrictByteString oracle) === ours
   )
     & counterexample ("value: " <> show x)
     & counterexample ("\n─── cborg: \n" <> CBOR.prettyHexEnc oracle)
-    & counterexample ("\n─── ours: \n" <> toString (encodeBase16 $ convert ours))
+    & counterexample ("\n─── ours: \n" <> toString (encodeBase16 $ Plutus.fromBuiltin ours))
  where
   oracle = encodeOracle x
   ours = encodingToBuiltinByteString (encodeOurs x)
 
 -- | Measure that the execution cost of encoding a certain value 'x' is small in
 -- front of some max execution budget.
 propCostIsSmall ::
-  Plutus.ToData a =>
-  Rational ->
+  (Plutus.ToData a, Show a) =>
+  (Double, Double) ->
   ExUnits ->
-  (a -> Encoding, Scripts.TypedValidator (EncodeValidator a)) ->
+  Scripts.TypedValidator (EncodeValidator a) ->
   a ->
   Property
-propCostIsSmall tolerance (ExUnits maxMemUnits maxStepsUnits) (encode, validator) a =
+propCostIsSmall (toleranceMem, toleranceStep) (ExUnits maxMemUnits maxStepsUnits) validator a =
```

Comment:
Suggestion (rename): `tolerance` seems to imply the results are within some bounds with some tolerance. Here it's more a `threshold` or an `uppperBound`